app/full_text_search.py

```diff
@@ -131,6 +131,22 @@ def create_jp_name_fields(**kwargs):
                         value=romanized_japanese_name)
                 )
                 romanized_names_list.append(romanized_japanese_name)
+                kwargs[field] = romanized_japanese_name
+
+    # field for searching by fullname without space
+    romanized_given_name = kwargs['given_name']
+    romanized_family_name = kwargs['family_name']
+    if romanized_given_name and romanized_family_name:
```

Comment:
As discussed offline, please apply this for unicode-based romanization too, so that it works for hiragana too.